lib/super_digit.rb

```diff
@@ -1,15 +1,16 @@
 # Superdigit
 
-# Time Complexity - ?
-# Space Complexity - ?
+# Time Complexity - O(n)
+# Space Complexity - O(n)
 def super_digit(n)
```

Comment:
Because `n.digits.sum` is O(n) and you're doing it with every recursive call, this is an O(n) in time complexity algorithm.